public void testGetCurrentPrescriptions() throws Exception {
    try {
        evilDAO.getExpiredPrescriptions(1);
        fail("DBException should have been thrown");
    } catch (DBException e) {
        assertEquals(EvilDAOFactory.MESSAGE, e.getSQLException().getMessage());
    }
}